lib/pf/access_filter/radius.pm

```diff
@@ -17,16 +17,32 @@ use warnings;
 
 use pf::security_event qw (security_event_view_top);
 use pf::locationlog qw(locationlog_set_session);
-use pf::util qw(isenabled generate_session_id);
+use pf::util qw(isenabled generate_session_id random_from_range extract);
+use List::MoreUtils qw(uniq);
 use pf::CHI;
 use pf::radius::constants;
 use Scalar::Util qw(reftype);
-use Number::Range;
+use pf::log;
 
 use base qw(pf::access_filter);
 tie our %ConfigRadiusFilters, 'pfconfig::cached_hash', 'config::RadiusFilters';
 tie our %RadiusFilterEngineScopes, 'pfconfig::cached_hash', 'FilterEngine::RadiusScopes';
 
+our %LOOKUP = (
+    session_id => \&setSession,
+);
+
+our %FUNCS = (
```

Comment:
Since this looks exactly like the switch templates functions, we should regroup the common ones together and reuse them in both places